### DISCUSSION TITLE
[Node Operator Question] After Pruning

### DISCUSSION THREAD

**Discussion by pikansj0s** [ethereum-optimism/developers#185] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

After pruning I get this error and have to restore from backup. Any ideas?

### Protocol Description

op-geth v1.101304.2
op-node v1.3.2
Chain ID  10
L1 chain eth geth version 1.13.5

### Node Logs

ERROR[01-11|11:22:11.523] Failed to create sealing context         err="missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found"
ERROR[01-11|11:22:11.523] Failed to build payload                  err="missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found"
WARN [01-11|11:22:11.523] Served engine_forkchoiceUpdatedV2        reqid=25954222 duration="833.799µs" err="Invalid payload attributes" errdata="{Error:missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found}"

**Comment by opfocus** [comment]
The point to confirm is that after pruning, it will be a snap mode node, and optimism does not seem to support snap synchronization mode. I am puzzled about this; I think it may not be prunable.   cc @sbvegan

**Reply by sbvegan** [reply]
I was under the impression that nodes could be pruned in their current state. Was your node stopped while pruning?

**Comment by sbvegan** [comment]
I have confirmed that offline pruing works in the same way as geth. So you can prune op-geth. You can refer to these geth docs as a reference: https://geth.ethereum.org/docs/fundamentals/pruning

**Reply by pikansj0s** [reply]
I can prune it, but the database is dead after pruning and resulting in error as shown above. Neither of you get the error, starting after pruning?

**Reply by 0xtakamaka** [reply]
How did you solve it without resyncing? 

I have got this error after I upgraded to v0.7.0 (base mainnet) stack today. 

WARN [02-19|16:46:43.228] Ignoring already known beacon payload    number=10,735,336 hash=a43e2c..3a95f5 age=1d4h6m
ERROR[02-19|16:46:43.236] Failed to create sealing context         err="missing trie node 9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 (path ) state 0x9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 is not available, not found"
ERROR[02-19|16:46:43.236] Failed to build payload                  err="missing trie node 9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 (path ) state 0x9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 is not available, not found"
WARN [02-19|16:46:43.236] Served engine_forkchoiceUpdatedV2        reqid=67532 duration=1.398933ms err="Invalid payload attributes" errdata="{Error:missing trie node 9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 (path ) state 0x9d087758857991e818c8961c741acac66a59fe9b2d59398b9d48f82e2dd5e7c4 is not available, not found}"

**Comment by cifer76** [comment]
Hi @sbvegan have we find any ways to fix this? I have a node (but forget whether or not did snap pruning before), after upgrading to v1.101308.2  op-geth report this error and cannot start
